osu.Game/Scoring/ScoreInfo.cs

```diff
@@ -49,7 +52,7 @@ public BeatmapInfo BeatmapInfo
         /// <summary>
         /// The <see cref="osu.Game.Beatmaps.BeatmapInfo.Hash"/> at the point in time when the score was set.
         /// </summary>
-        public string BeatmapHash { get; private set; } = string.Empty;
+        public string BeatmapHash { get; set; } = string.Empty;
```

Comment:
Do we really need to make the code more unsafe just to migration looking nice?

Reply:
The previous version (assigning property to its own backing property) is "clever" but actually horrible for usability.

This is why I didn't *say* to do this explicitly, I *asked* if it was feasible. An issue like this makes this approach stop being feasible. At this point if we *have* to go with this flow I don't know if I wouldn't prefer to go back to the old version because this just feels universally worse from a maintenance standpoint.